sceptre/cli/init.py

```diff
@@ -29,12 +30,12 @@ def init_stack_group(ctx, stack_group):
     Creates STACK_GROUP folder in the project and a config.yaml with any
     required properties.
     """
-    cwd = ctx.obj["sceptre_dir"]
-    for item in os.listdir(cwd):
+    context = SceptreContext(project_path=ctx.obj["project_path"])
```

Comment:
Ths needs updating now as we require two args

Reply:
removed altogether this func doesn't require context